connect_ext_ppr/models/deployment.py

```diff
@@ -63,15 +64,19 @@ class DeploymentRequest(Model):
     aborted_by = db.Column(db.String(20), nullable=True)
 
     ppr = relationship('PPRVersion', foreign_keys="DeploymentRequest.ppr_id")
-    deployment = relationship('Deployment', foreign_keys="DeploymentRequest.deployment_id")
+    deployment = relationship(
+        'Deployment',
+        foreign_keys="DeploymentRequest.deployment_id",
+        innerjoin=True,
+    )
 
 
 class MarketplaceConfiguration(Model):
     __tablename__ = 'marketplace_configuration'
 
     id = db.Column(db.Integer(), primary_key=True, autoincrement=True)
     marketplace = db.Column(db.String(16))
-    deployment = db.Column(db.ForeignKey(Deployment.id), nullable=True)
+    deployment_id = db.Column(db.ForeignKey(Deployment.id), nullable=True)
```

Comment:
hehe, somewhere I already saw, that it was switched from deployment to deployment_id 🤣